yaqd_wright/wright-continuous-filter-wheel.toml

```diff
@@ -10,8 +12,29 @@ bugtracker = "https://github.com/wright-group/yaqd-wright/issues"
 PyPI = "https://pypi.org/project/yaqd-wright"
 
 [config]
+motor.type = "int"
+motor.default = 1
+motor.doc = "Index for filter wheel motor" I'm not confident this is right...
+
+serial_port.type = "string"
+serial_port.default = "COM6" not actually sure about this...
+serial_port.doc = "Serial port that provides communication between filter wheel motor and computer"
+
+baud_rate.type = "int"
+baud_rate.default = 57600
+baud_rate.doc = "Communication speed of serial port"
+
+units.type = "float"
```

Comment:
Units are not actually truly allowed to be anything except "deg", should just be set in the code rather than in the toml.

If it were to be included here, the type would be "string", not "float"